Lesson21/Kadir/app.js

```diff
@@ -17,35 +17,64 @@ window.addEventListener('load',function(){
   // Show images function
   function showImage(index) {
     // Remove active class from all images
-    images.forEach(
-      // For every element in images array call a function that removes "active" class
-    )
+        images.forEach(element => {
+          element.classList.remove("active");
+        });
+
+        // indicators.forEach(el => {
+        //   el.classList.remove("active");
+        // });
+
+        while(indicators.length>0){
+          indicators[0].classList.remove("active");
+        }
+
+      images[index].classList.add("active");
+      indicators[index].classList.add("active");
+
+      images[index].classList.toggle("visible");
```

Comment:
Why do you need it? I think when you add active class, the visibility should be handled